conjur-api/Policy.cs

```diff
@@ -0,0 +1,45 @@
+﻿// <copyright file="Policy.cs" company="Conjur Inc.">
+//     Copyright (c) 2016 Conjur Inc. All rights reserved.
+// </copyright>
+// <summary>
+// Conjur Policy entity
+// </summary>
+namespace Conjur
+{
+    using System.IO;
+    using System.Net;
+
+    public class Policy : Resource
+    {
+        private readonly string path;
+
+        internal Policy(Client client, string name)
+                : base(client, Constants.KIND_POLICY, name)
+        {
+            this.path = $"policies/{WebUtility.UrlEncode(client.GetAccountName())}/{Constants.KIND_POLICY}/{WebUtility.UrlEncode(name)}";
+        }
+
+        /// <summary>
+        /// Loading a Conjur policy MAML stream structure
+        /// into given policy name, over REST POST request.
+        /// </summary>
+        /// <param name="policyContent">Stream valid MAML Conjur policy strature.</param>
+        /// <returns>Policy creation response as a stream</returns>
+        public Stream LoadPolicy(Stream policyContent)
+        {
+            WebRequest req = Client.AuthenticatedRequest(this.path);
+            req.Method = WebRequestMethods.Http.Post;
+            req.ContentLength = policyContent.Length;
+            policyContent.Seek(0, SeekOrigin.Begin);
+
+            using (Stream reqStream = req.GetRequestStream())
+            {
+                policyContent.CopyTo(reqStream);
+                using (Stream resStream = req.GetResponse().GetResponseStream())
+                {
+                    return resStream;
```

Comment:
You shouldn't use `using` with the result stream if you are going to return it directly. This is going to call `Dispose` on the stream when the method is returned, rendering the stream already closed when the caller gets it back. 

It would be good to add a test case that uses the result to make sure you get what you expect it. It looks like the current test calls `LoadPolicy` but ignores the result.